documentcloud/documents/admin.py

```diff
@@ -56,3 +58,30 @@ def save_model(self, request, obj, form, change):
 
     def delete_model(self, request, obj):
         obj.destroy()
+
+
+@admin.register(Entity)
+class EntityAdmin(admin.ModelAdmin):
+    """Entity Admin"""
+
+    list_display = ("name", "kind", "mid", "description", "wikipedia_url")
+    list_filter = ("kind", "description")
+    search_fields = ("name", "kind", "mid", "description", "wikipedia_url")
```

Comment:
This is a text search = I think name, mid and description make sense here (kind can be filtered by the `list_filter` options above, and searching through the wikipedia url does not make sense)

Reply:
If you have the wikipedia url and want to know if there's an entity for it, how else would you find it? I'll drop `kind`.